Reference/Security/Security-settings/index.md

```diff
@@ -0,0 +1,49 @@
+
+The settings for Umbraco passwords are handled by default through the ASP.NET Membership Providers. There are two Membership Providers in the web.config file. One for Umbraco Members and one for Users.
+
+    <membership defaultProvider="UmbracoMembershipProvider" userIsOnlineTimeWindow="15">
+      <providers>
+        <clear />
+        <add name="UmbracoMembershipProvider" type="Umbraco.Web.Security.Providers.MembersMembershipProvider, Umbraco" minRequiredNonalphanumericCharacters="0" minRequiredPasswordLength="10" useLegacyEncoding="false" enablePasswordRetrieval="false" enablePasswordReset="false" requiresQuestionAndAnswer="false" defaultMemberTypeAlias="Member" passwordFormat="Hashed" allowManuallyChangingPassword="false" maxInvalidPasswordAttempts="50"/>
+        <add name="UsersMembershipProvider" type="Umbraco.Web.Security.Providers.UsersMembershipProvider, Umbraco"  enablePasswordRetrieval="false" enablePasswordReset="false" requiresQuestionAndAnswer="false" allowManuallyChangingPassword="false" maxInvalidPasswordAttempts="100"/>
+      </providers>
+    </membership>
+
+For both of these Membership Providers you can specify the following settings (if the attribute is not specified in the web.config the default value is used):
+
+- minRequiredNonalphanumericCharacters (default 0): The number of minimal non-alphanumeric characters that are required for a password
+- minRequiredPasswordLength (default 10): The minimal length of a password
+- useLegacyEncoding (default 'false'): Projects that have been upgraded from a pre Umbraco v7.6 version will have this setting set to 'true' as an older password encoding mechanism was used. This setting should not be used for new projects.
+- passwordFormat (default 'Hashed', possible options 'Clear', 'Encrypted', 'Hashed'): You can specify the way a password should be stored in the database. The options ‘clear’ and ‘encrypted’ should not be used.
+- enablePasswordReset (default 'false'): This setting is not used
+- enablePasswordRetrieval (default 'false'): This setting is not used
+- requiresQuestionAndAnswer (default 'false'): This setting is not used
+- allowManuallyChangingPassword (default 'false'): This setting is not used
+- maxInvalidPasswordAttempts (default 5): The number of attempts before an user or member is locked out
+- passwordAttemptWindow (default: 10 (minutes)): This setting is not currently used
+- passwordStrengthRegularExpression: You could specify a regular expression for a password
+
+## Login screen
+
+### Locking of Users
+It's impossible to brute force the authentication on the login screen because after 'maxInvalidPasswordAttempts' the account of the user will be locked, and until that account is unlocked in the Users section, no attempt will succeed.
+
+### Password reset on login screen
+The settings for this feature are documented [here](../../Config/umbracoSettings/index.md#security) and [here](../../../Getting-Started/Backoffice/Login/index.md#password-reset)
```

Comment:
consider telegraphing where the here links go to in the wording of the text, so the user doesn't have to click both links to see what documentation you are referring to eg

The settings relevant to this feature are documented in the [Umbraco Settings Security Section]((../../Config/umbracoSettings/index.md#security)) and [Backoffice Login Password Reset Section] (../../../Getting-Started/Backoffice/Login/index.md#password-reset)

or something similar kind of wordings